pkgs/applications/misc/terminix/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, fetchurl, lib, zlib, atk, glib, dbus, gtk3, gnome3
+, hicolor_icon_theme, libX11, unzip, gdk_pixbuf, cairo
+, libXext, libsecret, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "terminix-${version}";
+  version = "1.3.5";
+
+  src = fetchurl {
+    url = "https://github.com/gnunn1/terminix/releases/download/${version}/terminix.zip";
```

Comment:
This is prebuilt binary which they provide
https://github.com/gnunn1/terminix/releases/download/1.3.5/terminix.zip

This is source code itself.
https://github.com/gnunn1/terminix/archive/1.3.5.tar.gz

I cant do anything about it.